R/affectations.R

```diff
@@ -1,22 +1,35 @@
-NB_SESSIONS <- c(
-    "FC_FIRE" = 3,
-    "EMIR" = 2,
-    "MICA" = 2
-)
+library(combinat)
+
+source("R/config.R")
+source("R/tables.R")
+
+local({ # Check that the two vectors are consistent
+    if (length(NB_SESSIONS) != length(SESSION_DEBUT)) {
+        stop("Erreur: Le nombre de sessions n'est pas le m\U00EAme entre NB_SESSIONS et SESSION_DEBUT.")
+    }
+    max_sessions <- max(NB_SESSIONS)
+    for (i in names(NB_SESSIONS)) {
```

Comment:
Using any() looks cleaner
```suggestion
    if (any(SESSION_DEBUT - 1 + NB_SESSIONS > max_sessions)) {
        stop(paste("Erreur: trop de sessions pour", names(which(SESSION_DEBUT - 1 + NB_SESSIONS > max_sessions))))
    }
```